sdk/storage/azblob/blob/models.go

```diff
@@ -100,8 +109,8 @@ func (o *DownloadToStreamOptions) format() (*generated.BlobClientDownloadOptions
 
 // ---------------------------------------------------------------------------------------------------------------------
 
-// DownloadToWriterAtOptions identifies options used by the DownloadToBuffer and DownloadToFile functions.
-type DownloadToWriterAtOptions struct {
+// DownloadWriterAtOptions identifies options used by the DownloadBuffer and DownloadFile functions.
+type DownloadWriterAtOptions struct {
```

Comment:
Since we're no longer exported the matching method for this, we should rename it to something simpler like `DownloadOptions`.

Reply:
It'll collide with the actual Download() function present in the package. So I think we can keep it like this.

Reply:
I'm not following, can you please clarify?

Reply:
There's already a DownloadOptions for Download() function. So renaming it DownloadOption will create a conflict. May be some other name perhaps?

Reply:
Ohh never mind. Apologies for the confusion.

Reply:
Renamed both the function and option bag - `download` `downloadOptions`